desktop/previous-versions/edge-releases-mac.md

```diff
@@ -2287,7 +2287,7 @@ events or unexpected unmounts.
 
 **Bug fixes and minor changes**
 
-* Documentation moved to [https://docs.docker.com/desktop/mac/](../index.md)
+* Documentation moved to [{{ site.docs_url }}/desktop/mac/](../index.md)
```

Comment:
Do we need `{{ site.docs_url }}` here? Wondering if it should be the same as other relative URLs on this page?

Reply:
I think here it's a implicitly showing the absolute url but agree that `/desktop/mac/` is enough, will fix that.